Documentation/Books/Manual/Upgrading/OSSpecificInfo/Windows.md

```diff
@@ -59,3 +59,22 @@ To use the old configuration, it is currently needed to:
 Manual upgrade of 'ZIP archive' installation
 --------------------------------------------
 
+There are two ways to upgrade a _Single Instance_ that has been started
```

Comment:
Nitpick: _started_ or _installed_? You can't run a zip file.

Reply:
I have now used " started from" should work..